### PR TITLE
Add a concepts/intercepts page with Brian's animation

### PR DESCRIPTION
**Pull request by LukeShu**
This is the counterpart to https://github.com/datawire/getambassador.io/pull/1163

### CODE REVIEW

**Comment by LukeShu** [comment]
v2: I've changed which browser extension we link to to set the headers

File: docs/pre-release/concepts/intercepts.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+by a developer on their laptop and the **<span style="color:
+#00c05b">green</span>** are made by a teammate on a different laptop.
+
+They can intercept the same service in the Kubernetes cluster to
+create a shared development environment.
+
+ 1. Intercept your service from your CLI:
+
+    ```shell
+    telepresence intercept SERVICENAME --http-match=Personal-Intercept=126a72c7-be8b-4329-af64-768e207a184b
```
Comment (donnyyung) [inline docs/pre-release/concepts/intercepts.md:115]: I would add a qualifier here saying "we're using `Personal-Intercept=126a72c7-be8b-4329-af64-768e207a184b` as the header, but you can use any <key>=<value> that you want" or something like that just so it's abundantly obvious that this value can, and should, change.

**Review by donnyyung — CHANGES_REQUESTED** [review]
This looks good just one suggestion to make it clear that the header you picked is just an example, since I could see people copying that command verbatim and thinking they need to use that specific http-header whenever they do a personal intercept.

**Review by donnyyung — APPROVED** [review]
LGTM thanks!

**Comment by LukeShu** [comment]
I clicked merge several hours ago.  I swear I did.  Why isn't it merged!?

**Pull request merged**